core/src/epicli/data/common/ansible/playbooks/roles/kafka/templates/kafka.service.j2

```diff
@@ -2,6 +2,12 @@
 Description=Kafka Daemon
 After=zookeeper.service
 
+{% if specification.kafka_var.javax_net_debug is defined %}
```

Comment:
`kafka_var` ? maybe we should refactor this sometime in the future ? :thinking:

Reply:
@sk4zuzu have you seen "Gopher's day" movie? ;)

Reply:
I don't know... :disappointed: